### DISCUSSION TITLE
[Tips for Windows] How to actually use it!

### DISCUSSION THREAD

**Discussion by Aruthuro** [houmain/keymapper#173] [Q&A] [answered]
If you get easily lost in the technical texts in the readme section (like me) and want it to work on windows, this might help you. You need to configure the application first so you can find the "configuration file" to actually map your keys, for that, open the **cmd on the installation folder** and run the "keymapperctl.exe" by typing its name on the cmd, them it gonna create the "config file" somewhere, in my case, it was on the user folder. Then just open the file and type the instructions you want to set, and when you save it, the app process 

1. You install it in your machine with the .msi file or the zip;
2. You open the folder were it was installer and run CMD on it;
3. Execute `keymapperctl.exe --set-config "file name"`;
4. The, just find were the "file" were created and configure it using some text editor. (It was created on the user fold for me).

**Comment by houmain** [comment, marked as answer]
Hi, thanks for the writeup and sorry, that it is not completely self-explanatory yet. I will try to further improve the README.

There is no need for using keymapperctl though. After installing the .msi one can click on the "Opening configuration file failed" notification or click the "Configuration" item in the tray icon menu to open the configuration file in the default editor.
